tests/unittests/load_functions/stub_http_trigger/__init__.py

```diff
@@ -0,0 +1,6 @@
+# This function app is to ensure the code outside main() function
+# should only get loaded once in __init__.py
+
+
+def main(req):
```

Comment:
This is used for testing backward compatibility in case some users are using `from __app__.StubTrigger.__init__ import something_in_trigger`.
The test file is in tests/unittests/load_functions/load_outside_main/main.py